dev/io.openliberty.java.internal_fat/src-reference/java18/io/openliberty/java/internal/TestApp.java

```diff
@@ -0,0 +1,18 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation and others.
```

Comment:
Please update the copyright date since this is a new file.

Reply:
Done